### ISSUE TITLE
Accessibility: Inspect should be focus on the entire listView item row, not the first item when ListView's FullRowSelect = true 

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4150]
* .NET Core Version: 
.Net SDK 6.0.100-alpha.1.20522.5

* Have you experienced this same bug with .NET Framework?: 
No

* More Info:
This is a regression issue, cannot repro in .Net Core 3.1

**Minimal repro:**
1. Open the attached WinFormsApp12.zip application in VS, then build and run it.
[WinFormsApp12.zip](https://github.com/dotnet/winforms/files/5426925/WinFormsApp12.zip)
2. Launch Inspect to focus on ListView items

**Actual behavior:**
Inspect focus on the first list item, not entire listView Item row when its FullRowSelect is true
![image](https://user-images.githubusercontent.com/26474449/96949258-14f70c80-151a-11eb-938c-2439884a2761.png)

**Expected behavior:**
Inspect focus on entire listView Item row when its FullRowSelect is true
![image](https://user-images.githubusercontent.com/26474449/96949344-44a61480-151a-11eb-8b8e-cde87b4a8f4a.png)
